src/decode.c

```diff
@@ -3016,7 +3015,7 @@ get_base_value_type(short gc)
               if (gc <= 469) return VT_REAL;
               if (gc <= 479) return VT_STRING;
               if (gc <= 998) return VT_INVALID;
-              if (gc == 999) return VT_STRING;
+              /*if (gc == 999)*/ return VT_STRING;
```

Comment:
why?

Reply:
Not a mistake. Just this comparison is always true